client/Fireport/Assets/Scripts/Model/State/AnimationAction.cs

```diff
@@ -3,7 +3,16 @@ namespace Model.State {
     public class AnimationAction {
         public int ID;
         public string Name;
-        public int Target;
-        public int Value;
+        public int Owner;
+        public float Time;
```

Comment:
I think this should be a part of the AnimationAction on the server.  That way we can control how long each animation runs for on the server and the client will just parse out whatever we give it.